# The original SLOOP: action spaces are low-level
# on top of a grid map

# Transition Model
transition_model: sloop_object_search.generic.SLOOPTransitionModel
robot_trans_model: default

# Observation Model
observation_model: sloop_object_search.basic.JointObservationModel

# Reward Model
reward_model: sloop_object_search.basic.RewardModel

# Policy Model
policy_model: sloop_object_search.basic.LowLevelPolicyModel2D

# Belief
belief_dist: sloop_object_search.basic.JointBelief
belief_params:
  init_belief: uniform
  object_belief: sloop_object_search.basic.LowLevelObjectBelief2D
  robot_belief: sloop_object_search.basic.ObservableRobotBelief2D

# Agent
agent: sloop_object_search.basic.SLOOPBasicAgent

# Planner
planner: pomdp_py.POUCT
